crypto/http/http_client.c

```diff
@@ -307,14 +343,14 @@ OSSL_HTTP_REQ_CTX
     }
     /* remaining parameters are checked indirectly by the functions called */
 
-    if ((rctx = OSSL_HTTP_REQ_CTX_new(wbio, rbio, maxline, max_resp_len, timeout,
```

Comment:
max_resp_len is now unused, should it be dropped?

Reply:
Good spotting; done.